src/app/features/tech-record/components/tech-record-summary/tech-record-summary.component.ts

```diff
@@ -112,12 +111,10 @@ export class TechRecordSummaryComponent implements OnInit {
       /*  7 */ PsvBrakeSectionWheelsHalfLocked,
       /*  8 */ PsvDdaTemplate,
       /*  9 */ DocumentsTemplate,
-      /* 10 */ getBodySection(VehicleTypes.PSV),
-      /* 11 */ getGrossVehicleWeightsSection(VehicleTypes.PSV),
-      /* 12 */ PsvTrainWeight,
-      /* 13 */ getAxleWeightsSection(VehicleTypes.PSV),
-      /* 14 */ getTyresSection(VehicleTypes.PSV),
-      /* 15 */ getDimensionsSection(VehicleTypes.PSV, this.vehicleTechRecord.noOfAxles, this.vehicleTechRecord.dimensions?.axleSpacing),
+      /* 10 */ PsvWeight,
+      /* 11 */ getBodySection(VehicleTypes.PSV),
```

Comment:
The order of the `Weights` and the `Body` sections has been reversed. Is this intentional? Unless the ticket mentions this specifically, the order should be respected.
```suggestion
      /* 10 */ getBodySection(VehicleTypes.PSV),
      /* 11 */ PsvWeight,
```